### ISSUE TITLE
[A11y][ARIA]Label name is not given at the control for the 'Select new member role' and 'change member role' dropdowns

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8193]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188120
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450) 

## Repro Steps:

    Change the Display resolution to 1280*1024 in System display settings
    Hit the URL and login with appropriate credentials
    Tab till "Open Profile drop down" and press enter
    Tab till "Manage organizations" and Press enter
    Tab till "Edit " present in any record the table and press enter
    Tab till Members and expand it 
    Tab till 'Select new member role' and verify the visual name is given or not

## Actual Results:
Label name is not given visually for the 'Select new member role' and 'change member role' dropdowns

## Expected Results:
Label name which is announcing by the screen reader for the Select new member role' and Change member role' should be given as visually. 
## User Impact:
Speech input users will face the issue as they may not be able to access this two dropdowns  

## Notes
Another aria-label fix. Should be fairly straightforward to fix. I don't believe the display resolution adjustment is necessary to repro.
